### DISCUSSION TITLE
Возможность отключить титулы

### DISCUSSION THREAD

**Discussion by ItzIsonn** [iBuseWinner/Reputation#16] [Ideas]
Предлагаю добавить возможность отключать титулы, а то если не хочешь использовать их, приходится ставить большие значения. 
По типу - enable_tituls: false

**Comment by iBuseWinner** [comment]
Будет реализовано в ближайшее свободное время.

**Comment by iBuseWinner** [comment]
Оставить поле пустым, чтобы убрать титулы. В этом случае выводиться будет пустой титул.
